index.d.ts

```diff
@@ -0,0 +1,47 @@
+declare module 'svelte-routing' {
+  import { SvelteComponent, SvelteComponentTyped } from 'svelte'
```

Comment:
There is no style guide or prettier config for the repo, but since all the js code uses semicolons, I think the types should too...

Reply:
I was trying to avoid them, but you have a point 😆